pex/bootstrap.py

```diff
@@ -41,14 +41,17 @@ def demote(self):
         """
         import sys  # Grab a hold of `sys` early since we'll be un-importing our module in this process.
 
+        # N.B.: We mutate the sys.path before un-importing modules so that any re-imports triggered
```

Comment:
Should this link as well to an issue here to provide additional context?

Reply:
Yup - will do.